packages/client/shared/tiptap/extensions/PageLinkBlockBase.ts

```diff
@@ -49,6 +50,16 @@ export const PageLinkBlockBase = Node.create({
             'data-canonical': attributes.canonical ? '' : undefined
           }
         }
+      },
+      isDatabase: {
+        parseHTML: (element) => {
+          return element.getAttribute('data-isDatabase') === '' ? true : false
+        },
+        renderHTML: (attributes) => {
+          return {
+            'data-isDatabase': attributes.isDatabase ? '' : undefined
```

Comment:
itty bitty nit here, but this is mixing kebab case with camel case. HTML & CSS usually take on just kebab case